pkg/scheduler/schedule_one_test.go

```diff
@@ -2660,6 +2662,73 @@ func TestZeroRequest(t *testing.T) {
 }
 
 func Test_prioritizeNodes(t *testing.T) {
+	podSpec := v1.PodSpec{
+		Containers: []v1.Container{
+			{
+				Image: "gcr.io/40",
+			},
+		},
+	}
+	pod1Spec := v1.PodSpec{
+		Containers: []v1.Container{
+			{
+				Image: "gcr.io/300",
+			},
+		},
+	}
```

Comment:
those specs are used once, so just inline them instead of defining those variables.